### DISCUSSION TITLE
Does attachment support need added to the Twitter client?

### DISCUSSION THREAD

**Discussion by alexpaden** [elizaOS/eliza#1504] [Q&A]
Hello. I was just wondering if the attachments were intentionally removed from memory or if adding something like https://github.com/elizaOS/eliza/commit/1ac20bc821e264ab199691c6c90d204bbcb51ac7 would be helpful.

tldr tweet wasn't putting attachments (photo/video/url Media) into memory/message

**Comment by themeshri** [comment]
did you find a solution? @alexpaden

**Reply by alexpaden** [reply]
oh yeah the commit i tagged is a working solution i just didn't know if u guys hadn't added it for a reason... @themeshri 

i'm checking out the new version now so maybe i'll just commit it to the repo if it's still not in here
